python/lsst/ts/externalscripts/auxtel/__init__.py

```diff
@@ -24,3 +24,4 @@
 from .latiss_acquire_and_take_sequence import *
 from .latiss_cwfs_align import *
 from .make_latiss_bias import *
+from .make_latiss_calibrations import *
```

Comment:
This looks like it supersedes the `make_X_bias` code.  Is that correct, and is the plan to have just the main multi-calibration script?

Reply:
That's correct.